packages/indy-sdk-to-askar-migration/package.json

```diff
@@ -25,28 +25,9 @@
   },
   "dependencies": {
     "@aries-framework/core": "0.3.3",
-    "@hyperledger/aries-askar-react-native": "0.1.0-dev.2",
-    "bs58": "5.0.0",
-    "buffer": "6.0.3",
-    "cbor": "8.1.0",
-    "class-transformer": "0.5.1",
-    "class-validator": "0.13.1",
-    "crypto-js": "4.1.1",
-    "react-native-fs": "2.20.0",
-    "react-native-sodium": "0.4.0",
-    "react-native-sqlite-storage": "^6.0.1",
-    "rxjs": "^7.2.0",
-    "sodium-javascript": "0.8.0",
-    "text-encoding-polyfill": "^0.6.7",
-    "tiny-msgpack": "2.0.2",
-    "uint8arrays": "^4.0.3"
```

Comment:
🚀

Reply:
Yeah that felt good :). Also, now we directly have node.js support.

Reply:
Ah didn't think about that. That is awesome